data-prepper-plugins/service-map-stateful/src/main/java/com/amazon/dataprepper/plugins/prepper/ServiceMapStatefulPrepper.java

```diff
@@ -10,13 +10,19 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.primitives.SignedBytes;
 import io.opentelemetry.proto.collector.trace.v1.ExportTraceServiceRequest;
+import org.apache.commons.lang3.tuple.Pair;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.File;
 import java.io.Serializable;
 import java.time.Clock;
-import java.util.*;
+import java.util.Collection;
+import java.util.Collections;
```

Comment:
might be some extra imports up here

Reply:
Both the Collection and Collections imports are needed in this class. Since we don't want any wildcard imports in our project, I configured my IntelliJ so it explicitly added these single exports.

Reply:
I see, you replaced the wildcard with the individual imports.